### PR TITLE
BZ-2092332 - Bug fix UEFI mode

### LINKED ISSUE #45753: [enterprise-4.10] Issue in file virt/virtual_machines/advanced_vm_management/virt-efi-mode-for-vms.adoc
### Which section(s) is the issue in?

[About EFI mode for virtual machines](https://docs.openshift.com/container-platform/4.10/virt/virtual_machines/advanced_vm_management/virt-efi-mode-for-vms.html)

### What needs fixing?

I think "EFI" should be "UEFI". "EFI" is the partition where the UEFI boot files are stored. I am fairly certain that "UEFI" is the industry term, not EFI, for the boot mode that replaced BIOS.

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.10+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2092332

Link to docs preview: http://file.rdu.redhat.com/sjess/BZ0092332/virt/virtual_machines/advanced_vm_management/virt-uefi-mode-for-vms.html

### CODE REVIEW

File: modules/virt-booting-vms-uefi-mode.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+// * virt/virtual_machines/advanced_vm_management/virt-uefi-mode-for-vms.adoc
 
 :_content-type: PROCEDURE
-[id="virt-booting-vms-efi-mode_{context}"]
-= Booting virtual machines in EFI mode
+[id="virt-booting-vms-uefi-mode_{context}"]
+= Booting virtual machines in UEFI mode
 
-You can configure a virtual machine to boot in EFI mode by editing the VM manifest.
+You can configure a virtual machine to boot in UEFI mode by editing the VM manifest.
```
Comment (sjhala-ccs) [inline modules/virt-booting-vms-uefi-mode.adoc:9]: s/VM manifest/`VirtualMachine` manifest

File: modules/virt-booting-vms-uefi-mode.adoc [hunk 2]
```diff
@@ -16,7 +16,7 @@ You can configure a virtual machine to boot in EFI mode by editing the VM manife
 
 . Create a YAML file that defines a VM object. Use the firmware stanza of the example YAML file:
```
Comment (sjhala-ccs) [inline modules/virt-booting-vms-uefi-mode.adoc:17]: Suggestion:

Edit or create a `VirtualMachine` manifest file. Use the `spec.firmware.bootloader` stanza to configure UEFI mode:

**Review by sjhala-ccs — COMMENTED** [review]
Your changes LGTM. There are a couple of things that I noticed in the existing text that need to be fixed. Thanks Sabrina!

Also, this PR should apply to 4.10 as well

**Comment by bergerhoffer** [comment]
The enterprise-4.12 label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.11. And any PR going into `main` must also target the latest version branch (enterprise-4.12).

If the update in your PR does NOT apply to version 4.12 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/virt-booting-vms-uefi-mode.adoc [hunk 3]
```diff
@@ -47,8 +47,8 @@ spec:
               secureBoot: true <2>
 #...
```
Comment (sjhala-ccs) [inline modules/virt-booting-vms-uefi-mode.adoc:48]: Can you please remove the `#` before the ellipsis (`...`)? Thank you.

**Review by sjhala-ccs — APPROVED** [review]

**Pull request merged**